MSAL/src/cache/MSALRefreshTokenCacheItem.m

```diff
@@ -100,8 +100,7 @@ - (void)initDerivedPropertiesFromJson
     _user = [[MSALUser alloc] initWithDisplayableId:self.displayableId
                                                name:self.name
                                    identityProvider:self.identityProvider
-                                                uid:self.clientInfo.uid
-                                               utid:self.clientInfo.utid
+                                     userIdentifier:self.clientInfo.userIdentifier
```

Comment:
same question here, do we still need refresh token cache item?

Reply:
removed